docs/exploration.md

```diff
@@ -0,0 +1,24 @@
+---
+id: exploration
+title: Avail Uncharted
+sidebar_label: Exploration Team
+slug: /exploration-team
+description:  An overview of exploration team and their project.
+
+
+---
+
+## Exploring Uncharted Horizons in Modular Blockchains
+
+  <p>Welcome to the home of <b><i>Avail Uncharted</i></b>. This is the heart of exploration within the Avail ecosystem. Our mission is to encourage exciting initiatives and collaborate closely with the community. Through collaboration, we aim to gain deeper insights into the needs of builders within our ecosystem and push beyond the horizons of what is possible.</p>
+
+## Active Projects
+| Project              | Description | Repository |               
```

Comment:
The "Project" column contains repository names. We should improve this. 

We can either rename "Project" to "Repository" and remove the existing "Repository" column, or keep "Project" but rename each entry into a more formal project name, and then use the entries that currently exist under "Project" to replace the usage of "link" in each row with the actual repository name. I think I'd be in favor of the latter.

Example:

```
| Adapter for Sovereign SDK |  A data availability adapter for sovereign rollups integrated with the Sovereign SDK   | [sovereign-da-adapter](https://github.com/availproject/sovereign-sdk/tree/main) |

```